draft-ietf-quic-recovery.md

```diff
@@ -216,23 +215,22 @@ implemented in QUIC.
 
 An unacknowledged packet is marked as lost when an acknowledgment is received
 for a packet that was sent a threshold number of packets (kReorderingThreshold)
-after the unacknowledged packet. Receipt of the ack indicates that a later
-packet was received, while kReorderingThreshold provides some tolerance for
-reordering of packets in the network.
-
-The RECOMMENDED initial value for kReorderingThreshold is 3.
-
-We derive this recommendation from TCP loss recovery {{?RFC5681}}
-{{?RFC6675}}. It is possible for networks to exhibit higher degrees of
-reordering, causing a sender to detect spurious losses. Detecting spurious
-losses leads to unnecessary retransmissions and may result in degraded
+and/or a threshold amount of time after the unacknowledged packet. Receipt of the
+ack indicates that a later packet was received, the reordering threshold
```

Comment:
Comma splice; I'd either re-add the conjunction ("while" seemed fine) or replace the comma with a semicolon.